app/controllers/api/v1x0/graphql_controller.rb

```diff
@@ -16,7 +16,7 @@ def query
       def overlay
         {
           "^.*$" => {
-            "base_query" => lambda do |model_class, _ctx|
+            "base_query" => lambda do |model_class, graphql_args, _ctx|
               policy_scope(model_class.all)
```

Comment:
`graphql_args` is not used.
We should switch to use `policy_scope(model_class)`, shouldn't we?

Reply:
yes, the change will be in the next PR.